deploy/ui/form.json

```diff
@@ -3088,6 +3088,30 @@
                   "description": "The image tag to use for Process Instance Migration e.g. 7.9.0, this param is optional for custom image.",
                   "type": "text"
                 },
+                {
+                  "label": "Process Instance Migration Admin Username, if empty the common AdminUser will be used.",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.username",
+                  "type": "text"
+                },
+                {
+                  "label": "Process Instance Migration Admin Password, if empty the common AdminPassword will be used.",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.password",
+                  "type": "password"
+                },
+                {
+                  "label": "Process Instance Migration ExtraClassPath Allows to add extra jars to the application classpath separated by colon. Needs to be mounted on the image before.",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.extraClassPath",
+                  "type": "text"
+                },
+                {
+                  "label": "Allows to append extra system properties to the PIM startup, only runtime system properties is supported.",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.processMigration.extraArgs",
+                  "type": "text"
+                },
```

Comment:
There is not property extraArgs, please remove this.

Looks like json ui form is not complete.

Reply:
good catch, forget to remove it.